File: app/src/main/java/net/tlalka/android/fiszki/model/db/DbManager.java
package net.tlalka.android.fiszki.model.db;

import android.content.Context;

import com.j256.ormlite.android.apptools.OpenHelperManager;

public class DbManager {

    public static DbHelper getHelper(Context context) {
        return OpenHelperManager.getHelper(context, DbHelper.class);
    }

    public static void releaseHelper() {
        OpenHelperManager.releaseHelper();
    }
}

File: app/src/main/java/net/tlalka/android/fiszki/model/db/DbManager.kt
package net.tlalka.android.fiszki.model.db

import android.content.Context

import com.j256.ormlite.android.apptools.OpenHelperManager

object DbManager {

    fun getHelper(context: Context): DbHelper {
        return OpenHelperManager.getHelper(context, DbHelper::class.java)
    }

    fun releaseHelper() {
        OpenHelperManager.releaseHelper()
    }
}
